src/portfolio.js

```diff
@@ -22,6 +22,8 @@ const socialMediaLinks = {
   gmail: "saadpasta70@gmail.com",
   gitlab: "https://gitlab.com/saadpasta",
   facebook: "https://www.facebook.com/saad.pasta7"
+  // medium: "https://www.medium.com/@<user name>",
+  // stackoverflow: "https://stackoverflow.com/users/<user id>/<user name>"
```

Comment:
Uncomment these and add these link 

https://medium.com/@saadpasta
https://stackoverflow.com/users/10422806/saad-pasta

Reply:
Will do!